src/custom/components/AffiliateStatusCheck/index.tsx

```diff
@@ -123,7 +123,7 @@ export default function AffiliateStatusCheck() {
       setAffiliateState('OWN_LINK')
 
       if (referralAddressQueryParam) {
-        history.push('/profile' + location.search)
+        history.push('/account' + location.search)
```

Comment:
I'd add a note here to update the path with @nenadV91 's changes where affiliate has it's own subpath now